ext/dom/php_dom.stub.php

```diff
@@ -787,7 +787,7 @@ public function validate(): bool {}
     /** @tentative-return-type */
     public function xinclude(int $options = 0): int|false {}
 
-    /** @return DOMNode|false */
+    /** @return DOMNode */
     public function adoptNode(DOMNode $node) {}
```

Comment:
please add a tentative return type here :)

Reply:
Thanks. I'll do that (if I get around to finishing this implementation :sweat_smile: )